datafusion/sql/src/planner.rs

```diff
@@ -1819,15 +1819,6 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 right: Box::new(self.sql_expr_to_logical_expr(*right, schema, ctes)?),
             }),
 
-
-            SQLExpr::UnaryOp { op, expr } => match (&op, expr.as_ref()) {
-                // The AST for Exists does not support the NOT EXISTS case so it gets
-                // wrapped in a unary NOT
-                // https://github.com/sqlparser-rs/sqlparser-rs/issues/472
```

Comment:
https://github.com/sqlparser-rs/sqlparser-rs/issues/472 is indeed fixed 🎉  (thanks @togami2864)